src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -0,0 +1,73 @@
+"""
+File       : MSRuleCleaner.py
+
+Description: MSRuleCleaner.py class provides the logic used to clean the Rucio
+block level data placement rules created by WMAgent.
+"""
+
+# futures
+from __future__ import division, print_function
+
+# system modules
+from retry import retry
+
+# WMCore modules
+from WMCore.MicroService.DataStructs.DefaultStructs import RULECLEANER_REPORT
+from WMCore.MicroService.Unified.MSCore import MSCore
+from Utils.EmailAlert import EmailAlert
+
+
+class MSRuleCleaner(MSCore):
+    """
+    MSRuleCleaner.py class provides the logic used to clean the Rucio
+    block level data placement rules created by WMAgent.
+    """
+
+    def __init__(self, msConfig, logger=None):
+        """
+        Runs the basic setup and initialization for the MSRuleCleaner module
+        :param msConfig: micro service configuration
+        """
+        super(MSRuleCleaner, self).__init__(msConfig, logger)
+
+        self.msConfig.setdefault("limitRequestsPerCycle", 500)
+        self.msConfig.setdefault("verbose", True)
+        self.msConfig.setdefault("interval", 60)
+        self.msConfig.setdefault("services", ['ruleCleaner'])
+        self.msConfig.setdefault("defaultGroup", "DataOps")
+        self.msConfig.setdefault("rucioAccount", RUCIO_ACCT)
+        self.uConfig = {}
```

Comment:
unified config and emailAlert are likely not going to be used in this MS

Reply:
agreed ... those I just missed to clean.

Reply:
Please remove `self.uConfig` in a future PR.